spec/models/article_spec.rb

```diff
@@ -16,34 +16,49 @@
 describe Article, type: :model do
 
   before(:each) do
-    @attr = { title: 'Titulo de ejemplo', text: 'Lorem ipsum dolor' + 
+    @user = FactoryGirl.create(:user)
+    @attr = { title: 'Titulo de ejemplo', text: 'Lorem ipsum dolor' +
     'sit amet, consectetur adipiscing elit, sed do eiusmod tempor'}
   end
 
   it "should create a new instance given valid attributes" do
-    Article.create!(@attr)
+    @user.articles.create!(@attr)
   end
 
   it "should have a title" do
-    no_title_article = Article.new(@attr.merge(title: ""))
+    no_title_article = @user.articles.build(@attr.merge(title: ""))
     expect(no_title_article).to be_invalid
   end
 
   it "should have text" do
-    no_text_article = Article.new(@attr.merge(text: ""))
+    no_text_article = @user.articles.build(@attr.merge(text: ""))
     expect(no_text_article).to be_invalid
   end
 
   it "should not have a text longer than 8000 chars" do
     long_text = "a" * 8000
-    long_text_article = Article.new(@attr.merge(text: long_text))
+    long_text_article = @user.articles.build(@attr.merge(text: long_text))
     expect(long_text_article).to be_invalid
   end
 
   it "should have a title less than 80 chars" do
     long_title = "a" * 100
-    long_title_article = Article.new(@attr.merge(title: long_title))
+    long_title_article = @user.articles.build(@attr.merge(title: long_title))
     expect(long_title_article).to be_invalid
   end
 
+  describe "user associations" do
+    before(:each) do
+      @article = @user.articles.build(@attr)
```

Comment:
Sure I mean is not wrong, but you can always create a factory with a related model. If you don't want all the users to be created with an article, you can always use "Traits":
https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits
and create something like: FactoryGirl.create(:user_with_article) and no need to add the @attr hash and the build line.